### PR TITLE
Feature 2384/k8s opabeat integration

### PR DESCRIPTION
**Pull request by DaveSys911**
Added support of k8s cis mounts to default agent manifests
## Why is it important?

To support add-on of cis integration to the existing agent as defined in [gh-#2384](https://github.com/elastic/security-team/issues/2384). The only difference between our yamls were the additional volume mounts added
## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

Deploy the agent (standalone & fleet managed) using the manifests in `deploy/kubernetes/elastic-agent-managed` & `deploy/kubernetes/elastic-agent-standalone` with kubebeat sub process enabled.

## Related issues

-

### CODE REVIEW

File: kubebeat/beater/kubebeat.go [hunk 1]
```diff
@@ -92,22 +92,30 @@ func (bt *kubebeat) Run(b *beat.Beat) error {
...
+				// ns will be passed as param from fleet on https://github.com/elastic/security-team/issues/2383 and it's user configurable
+				ns := ""
+				bt.resourceIteration(config.Datastream(ns), resource, runId, timestamp)
 			}
 
 			bt.scheduler.ScheduleResources(o, resourceCallback)
 		}
 	}
 }
+<<<<<<< HEAD
```
Comment (eyalkraft) [inline kubebeat/beater/kubebeat.go:104]: @DaveSys911 Seems like you merged things that don't compile! ⚠️ ⚠️

File: kubebeat/beater/evaluation_result_parser.go [hunk 2]
```diff
@@ -16,7 +19,7 @@ func NewEvaluationResultParser() (*evaluationResultParser, error) {
 	return &evaluationResultParser{}, nil
 }
 
-func (parser *evaluationResultParser) ParseResult(result interface{}, uuid uuid.UUID, timestamp time.Time) ([]beat.Event, error) {
+func (parser *evaluationResultParser) ParseResult(index, result interface{}, uuid uuid.UUID, timestamp time.Time) ([]beat.Event, error) {
```
Comment (oren-zohar) [inline kubebeat/beater/evaluation_result_parser.go:22]: breaking change :( how did it pass the tests?

Comment (DaveSys911) [reply kubebeat/beater/evaluation_result_parser.go:22]: This was before we had ci.

I confirmed this is working - can u  share what broke?

Comment (oren-zohar) [reply kubebeat/beater/evaluation_result_parser.go:22]: https://github.com/build-security/beats/blob/d139980d59c9392c0915af4626402e482c7393e7/kubebeat/beater/evaluation_result_parser_test.go#L20

Comment (DaveSys911) [reply kubebeat/beater/evaluation_result_parser.go:22]: Broken test,Broken Beat or both?

Missed the test as part of the review, but beat was working. Could u open a bug/task for this?

Comment (oren-zohar) [reply kubebeat/beater/evaluation_result_parser.go:22]: looks like only the test. I'll fix it in my next pr it's related

**Pull request merged**